.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,37 +5,26 @@ Summarize your PR. If it involves visual changes include a screenshot or gif.
 
 ### Checklist
 
-Delete any items that are not applicable to this PR.
+Check the PR satisfies following conditions. 
 
-- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)
-- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
+Reviewers should verify this PR satisfies this list as well.
+
+- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
 - [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
-- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
```

Comment:
CI already validates this.

Reply:
I would recommend to keep this:
- it helps to find the test runner link
- it reminds to people to run the flaky test runner in the first place

Reply:
Added back in https://github.com/elastic/kibana/pull/198617/commits/cca46c652fffed94b65fd1a16ad0b0a0c13bb3ee !